grails-app/views/customer/create.gsp

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <%-- <meta name="layout" content="main" /> --%>
```

Comment:
remove se não for utilizar

Reply:
vamos, mas ele ainda tá puxando o padrão do grails. o novo layout ainda ta com pr aberta